### DISCUSSION TITLE
How to create multiple alarms dedicated to each datetime?

### DISCUSSION THREAD

**Discussion by jm90hong** [gdelataillade/alarm#52] [General]
Hello!
I'm developing alarm app with this package.

I wanna create multiple 3 alarms such as 2023-05-20 11:00:00, 2023-05-21 11:00:00, 2023-05-22 11:00:00.
However, on 11:00:00 O'clock  3 notifications concurrently occured.

How could i deal with it??

**Comment by gdelataillade** [comment]
Hi @jm90hong,

It looks like there's an issue with the notifications. I'll take a look when I find some time. Feel free to contribute if you are interested!

**Comment by gdelataillade** [comment]
Hi @jm90hong,

You were right, the notification schedule date was broken when an alarm with a date that is more than 24 hours away was selected.

Sorry for my late answer. Please update the plugin to `1.1.2` and let me know if it works as expected.

Best regards
